external-crates/move/crates/move-cli/src/base/coverage.rs

```diff
@@ -40,7 +44,18 @@ pub enum CoverageSummaryOptions {
         module_name: String,
     },
     #[clap(name = "lcov")]
-    Lcov,
+    Lcov {
+        /// Compute differential coverage for the provided test name. Lines that are hit by this
+        /// test only will show as covered, and lines that are hit by both this test and all other
+        /// tests will show as "uncovered". Otherwise lines are not annotated with coverage
+        /// information.
+        #[clap(long = "differential-test")]
+        differential: Option<String>,
+        /// Compute coverage for the provided test name. Only this test will contribute to the
+        /// coverage calculation.
+        #[clap(long = "only-test", conflicts_with = "differential")]
```

Comment:
Should this be an enum? Or is that more trouble than it is worth?

Reply:
I looked into it and it's possible, but I'd classify it as more trouble than it's worth and leads to a (IMHO) slightly weird command texture where `differential` and `only-test` become subcommands as opposed to args/flags.